content/learn/video.html

```diff
@@ -0,0 +1,199 @@
+                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                allowfullscreen></iframe>
+        </div>
+        <div class="paragraph">
+            <p>KIE Live is a series of public live streamings for knowledge sharing about any Business Automation topic:
+                business rules, decisions, processes, resource planning, tooling, engine, trusty AI, etc. It is a
+                community effort, therefore, it mentions not only the enterprise products and features but also, the
+                projects that compose Red Hat Process Automation Manager and Decision Manager. It highlights features
+                that might be on the roadmap but are not yet supported in the products.</p>
+        </div>
+    </div>
+</div>
+<div class="sect1">
+    <h2 id="_kiedrop">KIE Drop</h2>
+    <div class="sectionbody">
+        <div class="justify-content-center text-center">
+            <iframe width="560" height="315"
+                src="https://www.youtube.com/embed/videoseries?list=PLo3ZScdD9hW5agTmsP-_lMz0zzhX_J0KM"
+                title="YouTube video player" frameborder="0"
+                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                allowfullscreen></iframe>
+        </div>
+        <div class="paragraph">
+            <p>A KIE Drop is a short, focused, five minute (or less) video used to promote an idea, one of our KIE
+                projects, or some feature in a project. Do you have an idea for a KIE Drop? Get in contact with us!</p>
+        </div>
+    </div>
+</div>
+<div class="sect1">
+    <h2 id="_kieweek">KIE Week 2021</h2>
+    <div class="sectionbody">
+        <div class="justify-content-center text-center">
+            <iframe width="560" height="315"
+                src="https://www.youtube.com/embed/videoseries?list=PLo3ZScdD9hW6UtFRbgOfYbdrcPkIjatQr"
+                title="YouTube video player" frameborder="0"
+                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                allowfullscreen></iframe>
+        </div>
+        <div class="paragraph">
+            <p>KIE Week series -- Exploring Process & Business Automation -- where we’ll take you on a general technical
+                and cultural journey through approaches to integrating process and business automation rules. You’ll
+                delve into workflow through the eyes of some of our core international Red Hat® engineers’ community
+                KIE/Drools projects. And, we’ll round out the week with a roundtable discussion about real-world
+                customer implementations of the Red Hat products built from KIE projects.
+                <br />
+                Our KIE Week will offer a single one-hour session each day. Join that one that especially resonates with
+                you or register for them all! </p>
+        </div>
```

Comment:
```suggestion
       </div>
        <div class="paragraph">
            <p>A series exploring process & business Automation, where we take you on a general technical
                and cultural journey through approaches to integrating process and business automation rules. You can
                delve into workflow through the eyes of some of our core international Red Hat® engineers’ community
                KIE projects. 
                <br />
                This KIE Week will offer a single one-hour session each day. </p>
        </div>
        <div class="justify-content-center text-center">
            <iframe width="560" height="315"
                src="https://www.youtube.com/embed/videoseries?list=PLo3ZScdD9hW6UtFRbgOfYbdrcPkIjatQr"
                title="YouTube video player" frameborder="0"
                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
                allowfullscreen></iframe>
        </div>        
```

Reply:
I do believe since it red-highlighted only row 79 in old, it introduced a dup.
Will remove the dup manually _assuming_ you meant this green is the new content for this section.